tests/integration/aps/test_utils.py

```diff
@@ -50,7 +50,6 @@ def test_dag_run(dag: DAG):
     repo = APSRepository()
     repo.delete_all()
     assert len(repo.find_all()) == 0
-    dag.clear()
```

Comment:
why we have to remove it?

Reply:
it was crashing on Github actions. Cannot remember the exact error, but the github action log just froze, if I wanted to see the traceback.

Reply:
crashing github actions? This should not be related maybe there was an outage. We shouldn't just delete things without understanding the issue :)

Reply:
M... put back `dag.clean()`, the error haven't been raised again